src/os/bluestore/NVMEDevice.cc

```diff
@@ -281,13 +281,6 @@ class SharedDriverData {
     registered_devices.swap(new_devices);
     _aio_start();
   }
-
-  uint64_t get_block_size() {
-    return block_size;
-  }
-  uint64_t get_size() {
-    return size;
-  }
```

Comment:
Revert this. Don't remove these functions. Because SharedDriverData doesn't inherit `BlockDevice` and so creates a different object.